planex/depend.py

```diff
@@ -167,7 +181,9 @@ def main():
                             defines=macros)
             pins[os.path.basename(pin_path)] = spec
 
-    for spec_path in args.specs:
+    links = {pkgname(lnk): lnk for lnk in args.specs if lnk.endswith(".lnk")}
+
+    for spec_path in [spec for spec in args.specs if spec.endswith(".spec")]:
```

Comment:
I am fine enforcing lowercase extensions. Is this the case with all our specfiles?